app/uk/gov/hmrc/fhregistrationfrontend/views/govuk_wrapper.scala.html

```diff
@@ -81,38 +81,12 @@
     <script type="text/javascript" src='@routes.Assets.at("javascripts/details.polyfill.js")'></script>
 }
 
-@analyticsGA = {
-    <script type="text/javascript">
-            (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-                (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o), m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a, m)
-            })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
-
-            ga('create', '@{appConfig.analyticsToken}', 'auto');
-
-            ga('create', '@{appConfig.analyticsGovUkToken}', 'auto', 'govuk_shared', {'allowLinker': true});
-            ga('govuk_shared.require', 'linker');
-            ga('govuk_shared.linker.set', 'anonymizeIp', true);
-            ga('govuk_shared.linker:autoLink', ['www.gov.uk']);
-            ga('govuk_shared.send', 'pageview');
-    </script>
-  }
-
 @scripts = {
 @polyfills
 @scriptElem
 }
 
-@bodyEnd = {
-@{
-    viewHelpers.footer(
-        analyticsToken = Some(appConfig.analyticsToken),
-        analyticsHost = appConfig.analyticsHost,
-        ssoUrl = None,
-        scriptElem = Some(scripts),
-        gaCalls = None
-    )
-}
-}
+@bodyEnd = {}
```

Comment:
Apparently there was a [second PR](https://github.com/hmrc/cbcr-frontend/pull/503/files), we do not entirely remove this, but replace it with an emptier one.

Reply:
@lutnos yes I have gone through both PR's earlier, but here it should be removed complete footer section otherwise save/submit buttons are not working(apparently we can see some errors, either we should keep as is other wise to be removed as above)